analyticsdata/src/get_common_metadata.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Google Analytics Data API sample application retrieving dimension and metrics
+ * metadata.
+ * See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/getMetadata
+ * for more information.
+ * Usage:
+ *   composer update
+ *   php get_common_metadata.php
+ */
+
+namespace Google\Cloud\Samples\Analytics\Data;
+
+// [START analyticsdata_get_common_metadata]
+use Google\Analytics\Data\V1beta\BetaAnalyticsDataClient;
+use Google\Analytics\Data\V1beta\Metadata;
+use Google\ApiCore\ApiException;
+
+/**
+ * Retrieves dimensions and metrics available for all Google Analytics 4 properties.
+ */
+function get_common_metadata()
+{
+    // Create an instance of the Google Analytics Data API client library.
+    $client = new BetaAnalyticsDataClient();
+
+    /*
+    * Set the Property ID to 0 for dimensions and metrics common
+    * to all properties. In this special mode, this method will
+    * not return custom dimensions and metrics.
+    */
+    $propertyId = 0;
+    $formattedName = 'properties/' . $propertyId . '/metadata';
```

Comment:
Let's explicitly convert the int to a string
```suggestion
    $formattedName = sprintf('properties/%s/metadata', $propertyId);
```